Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -33,6 +33,7 @@ public class NetworkServer : MonoBehaviour
 
         public NetworkConnectionEvent Connected = new NetworkConnectionEvent();
         public NetworkConnectionEvent Authenticated = new NetworkConnectionEvent();
+        public UnityEvent OnStopServer = new UnityEvent();
```

Comment:
rename it to "Stopped" to be consistent with the other events

Reply:
fixed